sky/cloud_stores.py

```diff
@@ -294,10 +391,20 @@ def make_sync_file_command(self, source: str, destination: str) -> str:
         return self.make_sync_dir_command(source, destination)
 
 
+def get_storage_from_path(url: str) -> CloudStorage:
+    """Returns a CloudStorage by identifying the scheme:// in a URL."""
+    result = urllib.parse.urlsplit(url)
+    if result.scheme not in _REGISTRY:
+        assert False, (f'Scheme {result.scheme} not found in'
+                       f' supported storage ({_REGISTRY.keys()}); path {url}')
+    return _REGISTRY[result.scheme]
+
+
 # Maps bucket's URIs prefix(scheme) to its corresponding storage class
 _REGISTRY = {
     'gs': GcsCloudStorage(),
     's3': S3CloudStorage(),
     'r2': R2CloudStorage(),
     'cos': IBMCosCloudStorage(),
+    'https': AzureBlobCloudStorage()
```

Comment:
We are now using the raw container url instead of the `az://` I came up with.

Reply:
I see. This feels a little bit strange to me, as it is possible to have other storage also using https links. Would this be better? Though not sure if it would be too complicated.

```python
az://https://xxx.xxx.xxx.xxx
```